plugins/ssl/cert_validity

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# check the certificate validity of your certfificates
+#
+# andreas perhab - andreas.perhab@wt-io-it.at
+# https://www.wt-io-it.at/
+#
+# License: GPLv2
+#
+# Installing: Add list of your certificates prefixed by the type in munin plugin-conf.d
+#
+# For openvpn ca.crt and crl.pem
+# [cert_validity]
```

Comment:
How about using a name, that indicates the *local* access to certificates (e.g. `certificate_file_expiry`)?

Reply:
renamed plugin to certificate_file_expiry